mitosheet/src/mito/components/elements/MultiToggleColumns.tsx

```diff
@@ -25,46 +28,63 @@ interface MultiToggleColumnsProps {
 const MultiToggleColumns = (props: MultiToggleColumnsProps): JSX.Element => {
 
     const columnIDsMap = props.sheetData?.columnIDsMap || {};
+    const [dismissedWarning, setDismissedWarning] = React.useState(false);
     const columnIDsAndDtype: [ColumnID, string][] = Object.entries(props.sheetData?.columnDtypeMap || {});
     const columnIDs: ColumnID[] = columnIDsAndDtype.map(([cid, ]) => {return cid});
 
     return (
-        <MultiToggleBox
-            searchable
-            onToggleAll={(newSelectedIndexes) => {
-                const newSelectedColumnIDs = newSelectedIndexes.map(index => {return columnIDs[index]});
-                props.onChange(newSelectedColumnIDs);
-            }}
-            height='medium'
-        >
-            {columnIDsAndDtype.map(([columnID, columnDtype], index) => {
-                const columnHeader = columnIDsMap[columnID];
+        <div>
+            {(props.warning !== undefined && !dismissedWarning) &&
+                <div className='caution-text-container'>
```

Comment:
I guess we don't need to use the full expandable component here?

Reply:
But if it's pretty much the same, should we just factor it into a separate file and fully separate component?

Reply:
I was thinking that - we could add params for these but there are a couple differences. One is that the wording for multiple invalid keys is hard-coded into the expandable component. There's also slight differences in the widths of the icons because the key warning is nested into a more narrow component than the columns multi toggle. Up to you!